recipe/conda_build_config.yaml

```
@@ -2,9 +2,27 @@ c_compiler_version:        # [linux]
   - 12                     # [linux]
 cxx_compiler_version:      # [linux]
   - 12                     # [linux]
-arm_variant_type:          # [aarch64]
-  - sbsa                   # [aarch64]
+arm_variant_target:
+  - None
+  - tegra                  # [linux]
+  - sbsa                   # [linux]
```

Can we swap the order here?

Given `tegra` is newly added, think we should append it. Also this will make it easier to keep the `None` (Windows) and `sbsa` (Linux ARM for servers) straight

```suggestion
arm_variant_target:
  - None
  - sbsa                   # [linux]
  - tegra                  # [linux]
```

---

done